package.json

```diff
@@ -107,7 +113,7 @@
     "auto-launch": "5.0.5",
     "electron-dl": "1.12.0",
     "electron-log": "2.2.17",
-    "electron-spellchecker": "1.1.2",
+    "electron-spellchecker": "git+https://github.com/KiranNiranjan/electron-spellchecker.git#v3.0.5",
```

Comment:
hope this is temporary.  shouldn't be maintaining forks in personal repo's - this will break build if we lose access to someones personal repo.

Reply:
@lneir Yes, This is temporary.